client/src/views/users/ActiveUsersPage.vue

```diff
@@ -494,11 +500,9 @@ onUnmounted(async () => {
   overflow-y: auto;
 }
 
-.users-toolbar {
-  padding: 1em 2em;
-  height: 6em;
-  background-color: var(--parsec-color-light-secondary-background);
-  border-top: 1px solid var(--parsec-color-light-secondary-light);
+.right-side {
+  margin-left: auto;
+  display: flex;
```

Comment:
are these changes specific to toast changes?

Reply:
Absolutely not, probably a stash applied to a wrong branch, thanks for noticing